src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.html

```diff
@@ -184,9 +184,13 @@
                               [selectionLimit]="4"
                               (selection)="appSelection()">
             </cd-select-badges>
+            <i *ngIf="data.applications.selected <= 0"
```

Comment:
```suggestion
            <i *ngIf="data.applications.selected <= 0"
```
@alfonsomthd is a cautious guy, right!? 😆 I said that because of the `<` there, but probably it's a healthy approach.